org.mixedrealitytoolkit.input/Interactors/GazePinch/GazePinchInteractor.cs

```diff
@@ -28,12 +28,43 @@ public class GazePinchInteractor :
         [Tooltip("The hand controller used to get the selection progress values")]
         private ArticulatedHandController handController;
 
+        #region Properties and methods from deprecated XRBaseController
+
+        XRControllerState m_ControllerState;
+        /// <summary>
+        /// The current state of the controller.
+        /// </summary>
+        public XRControllerState currentControllerState
+        {
+            get
+            {
+                SetupControllerState();
+                return m_ControllerState;
+            }
+
+            set
+            {
+                m_ControllerState = value;
+                m_CreateControllerState = false;
+            }
+        }
+
+        bool m_CreateControllerState = true;
+
+        void SetupControllerState()
+        {
+            if (m_ControllerState == null && m_CreateControllerState)
+                m_ControllerState = new XRControllerState();
+        }
+
+        #endregion
+
         /// <summary>
         /// Is the hand ready to select? Typically, this
         /// represents whether the hand is in a pinching pose,
         /// within the FOV set by the aggregator config.
         /// </summary>
-        protected bool PinchReady => handController.PinchSelectReady;
+        protected bool PinchReady => (currentControllerState is ArticulatedHandControllerState handControllerState) && handControllerState.PinchSelectReady;
```

Comment:
Pinch ready is the only state needed here.   Let's just store the "bool" value, and update PinchReady in this interactor.

Reply:
Done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/743/commits/07d41155987a3c194dbeb0c7404455a797418ada , resolving.